api/v1alpha1/releaseplan_types.go

```diff
@@ -51,17 +57,28 @@ type ReleasePlanSpec struct {
 	Target string `json:"target"`
 }
 
+// PairedReleasePlanAdmission defines the relevant information for a paired ReleasePlanAdmission.
+type PairedReleasePlanAdmission struct {
+	// Name contains the namespaced name of the releasePlanAdmission
+	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?\/[a-z0-9]([-a-z0-9]*[a-z0-9])?$
+	// +optional
+	Name string `json:"name,omitempty"`
+
+	// Active indicates whether the ReleasePlanAdmission is set to auto-release or not
+	// +optional
+	Active bool `json:"active,omitempty"`
```

Comment:
I added it because I think it is nice and would not be surprised at all if it was asked for in the future, especially as there are many ReleasePlans for one RPA

Reply:
Your code is not currently supporting this as you ignore label updates in your predicate I think. And it's not needed today. I'd remove it. Less reconciliations.

Reply:
I like it. I should fix the reconcile logic then. I think it is very nice to see. We can talk about it tomorrow. Left it for now